theHarvester/discovery/bevigil.py

```diff
@@ -0,0 +1,37 @@
+from theHarvester.lib.core import *
+
+class SearchBeVigil:
+    
+    def __init__(self, word):
+        self.word = word
+        self.totalhosts = set()
+        self.interestingurls = set()
+        self.key = Core.bevigil_key()
+        self.proxy = False
+
+    async def do_search(self):
+        subdomainEndpoint = f"https://osint.bevigil.com/api/{self.word}/subdomains/"
+        urlEndpoint = f"https://osint.bevigil.com/api/{self.word}/urls/"
+        headers = {
+            'X-Access-Token': self.key
+        }
```

Comment:
```suggestion
        headers = {'X-Access-Token': self.key}
```
Just to remove un-needed space being taken up